docs/platforms/javascript/guides/sveltekit/manual-setup.mdx

```diff
@@ -467,47 +517,31 @@ export const load = wrapServerLoadWithSentry(({ event }) => {
 });
 ```
 
-#### Configure Client-side `fetch` Instrumentation
+#### Instrument Server Routes
 
 <Note>
 
-Available since version `7.91.0`
+Available since `8.24.0`
```

Comment:
Reminder for myself to double check that this is the actual version

Reply:
seems like it's [8.25.0](https://github.com/getsentry/sentry-javascript/pull/13284)